Documentation/MigrateToComposer/BestPractices.rst

```diff
@@ -94,14 +100,99 @@ other composer packages.
 Don't forget to commit your updated :file:`composer.lock` to your version
 control system.
 
+.. note::
+
+    Please be sure to disable extensions in TYPO3's Extension Manager, before removing them with composer.
+    Or ensure to regenerate your :file:`typo3conf/PackageStates.php` file automatically, after removing the packages. You could use the
+    `extension "typo3_console" <https://docs.typo3.org/typo3cms/extensions/typo3_console/CommandReference/Index.html#install-generatepackagestates>`__ for that
+
 Check for available updates
 ===========================
 
 Run ``composer outdated`` to see a list of available updates.
 
+Completely clear "typo3conf/ext" folder
+=======================================
+
+In the "Migration Steps" chapter, this tutorial explained, how you can
+`keep your individual extension in "typo3conf/ext" <./MigrationSteps.html#include-individual-extensions-like-sitepackages>`__
+and in the "Co-working" section, there was a part about how to add rules
+to your :file:`.gitignore` file to exclude :file:`typo3conf/ext` from,
+but keep your individual extensions in Git.
+
+If you are searching for a solution to keep your :file:`typo3conf/ext` folder
+clean and unify the extension handling even for your project's individual
+extension, this chapter might be useful.
+
+Define a local path repository
+------------------------------
+
+Create a directory :file:`packages` in your project root folder and define
+this folder as a repository of type "path" in your :file:`composer.json`::
+
+    {
+        "repositories": [
+            {
+                "type": "path",
+                "url": "./packages/*"
+            }
+        ]
+    }
+
+Include your individual extensions from "packages" folder
+---------------------------------------------------------
+
+In the next step, you move all your individual extensions from
+:file:`public/typo3conf/ext` to :file:`packages`. And for this way to include them,
+it's important, that each extension has it's own correct :file:`composer.json` file.
+How this file should look in your extension, can be found on `composer.typo3.org <https://composer.typo3.org/>`__ or
+`this blog post from Helmut Hummel <https://insight.helhum.io/post/148886148725/composerjson-specification-for-typo3-extensions>`__.
+
+Assumed, your package key is, ``foo/bar``, you can type the following command to include your extension to your project::
+
+    composer require foo/bar:@dev
+
+In this case, it's the easiest way to not define any composer version
+number, but tell composer to use the latest ``dev``state.
```

Comment:
There has to be a whitespace after the last ` sign and before "state".
Otherwise the signs will be displayed instead of being applied.

Reply:
Thanks. I fixed it.